### PR TITLE
10 setup global styles and variables

### PR DESCRIPTION
**Pull request by younusk-hub**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,8 +8,13 @@
     "@testing-library/user-event": "^13.5.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+<<<<<<< HEAD
```
Comment (Charlie-robin) [inline package.json:11]: Remove the the merge conflict notes. 

```
<<<<<< HEAD
```

File: package.json [hunk 2]
```diff
@@ -8,8 +8,13 @@
...
+    "react-scripts": "5.0.1",
+    "sass": "^1.53.0",
```
Comment (Charlie-robin) [inline package.json:13]: `sass` should be a dev dependancy

File: src/assets/sass/_variables.scss [hunk 3]
```diff
@@ -0,0 +1,23 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@500&display=swap');
+
+// COLORS
+$primary-color: #B4DDC0;
```
Comment (Charlie-robin) [inline src/assets/sass/_variables.scss:4]: I am not sure about these names, perhaps something more descriptive?

```scss
$primary-color-green : #B4DDC0;
```
Perhaps we group them similar to how Jake has on the design system page.

image

File: src/assets/sass/_variables.scss [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+$quaternary-color: #2F2D53;
+$quinary-color: #AFCAC5;
+$senary-color: #2C2C2C;
+
+
+$white-color: white;
+$black-color: black;
+
+// FONTS
+
```
Comment (Charlie-robin) [inline src/assets/sass/_variables.scss:16]: Let's add some sizes for fonts.

image

File: src/App.scss [hunk 5]
```diff
@@ -0,0 +1,38 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
```
Comment (Charlie-robin) [inline src/App.scss:5]: Remove these styles

**Review by Charlie-robin — CHANGES_REQUESTED** [review]

**Review by nologyRob — APPROVED** [review]

**Pull request merged**